### PR TITLE
Update CLDR to version 37

### LINKED ISSUE #11303: Update CLDR to version 37
CLDR 37 has been released in [fujiwarat/cldr-emoji-annotation](https://github.com/fujiwarat/cldr-emoji-annotation) repository.

@leonardder, can you update it?

### PR DESCRIPTION
**Pull request by dpy013**
### Link to issue number:
fixes #11303  
### Summary of the issue:
Update CLDR to version 37
### Description of how this pull request fixes the issue:
Synchronize the latest official CLDR version
### Testing performed:
The local environment has been used to compile and test, and the build is completed.
### Known issues with pull request:
none
### Change log entry:

Section: Changes
- Update CLDR to version 37. (#11303)
thanks

### CODE REVIEW

**Review by OzancanKaratas — CHANGES_REQUESTED** [review]
Please update the readme reference.

After updating, this pull request can be merged and fixes #11303.

**Comment by josephsl** [comment]
Hi, for whoever is going to merge this, I think it would be best to cherry-pick rather than a squash merge due to what’s new edits. Thanks.

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -13,6 +13,7 @@ What's New in NVDA
 == Changes ==
 - The Report formatting script (NVDA+f) has now been changed to report the formatting at the system caret rather than at the review cursor position. To report formatting at the review cursor position now use NVDA+shift+f. (#9505)
 - NVDA no longer automatically sets the system focus to focusable elements by default in browse mode, improving performance and stability. (#11190)
+- Update CLDR to version 37. (#11303)
```
Comment (leonardder) [inline user_docs/en/changes.t2t:16]: Please revert this change. What's new is dealt with by NV Access at merge time.

```suggestion
```

**Review by LeonarddeR — DISMISSED** [review]

**Review by feerrenrut — APPROVED** [review]
This seems fine to me, the actions from @leonardder have been addressed.

**Pull request merged**